recipes/openblas/all/conanfile.py

```diff
@@ -52,8 +54,20 @@ def _configure_cmake(self):
 
         self._cmake.definitions["NOFORTRAN"] = not self.options.build_lapack
         self._cmake.definitions["BUILD_WITHOUT_LAPACK"] = not self.options.build_lapack
+
+        # This is a workaround to fix building lapack error
+        if self.options.build_lapack
+            self._cmake.definitions["CMAKE_Fortran_USE_RESPONSE_FILE_FOR_OBJECTS"] = self.options.build_lapack
         self._cmake.definitions["DYNAMIC_ARCH"] = self.options.dynamic_arch
         self._cmake.definitions["USE_THREAD"] = self.options.use_thread
+        if self.settings.compiler != "gcc":
+            self._cmake.definitions["USE_OPENMP"] = False
+            self.output.warn("Compiler does not support OpenMP.")
```

Comment:
most compilers support openmp, why gcc only here? Moreover build() is not the good place to display such warnings.